pegasus/sites.v3/code.org/public/css/tools.css

```diff
@@ -0,0 +1,71 @@
+#resources {
+  margin-top: 40px;
+}
+
+.tools-info {
+  border: 1px solid;
+  border-color: #0094ca;
```

Comment:
I've never been able to figure out a way to import color variables here so for now they will match the other css files and just use the numbers.